lib/DxilContainer/DxcContainerBuilder.cpp

```diff
@@ -9,12 +9,13 @@
 //                                                                           //
 ///////////////////////////////////////////////////////////////////////////////
 
+#include "dxc/Support/Global.h"
+#include "dxc/Support/WinIncludes.h"
```

Comment:
This move looks appropriate, but I don't see anything that would have made it necessary. Is it just an incidental cleanup?

Reply:
It is for DxilHash.h included in DxcContainerBuilder.h.
I've moved this change there.